src/lib/y2storage/fake_device_factory.rb

```diff
@@ -566,7 +556,7 @@ def create_lvm_lv(parent, args)
       size = args["size"] || DiskSize.zero
       raise ArgumentError, "\"size\" missing for lvm_lv #{lv_name}" unless args.key?("size")
 
-      lvm_lv = parent.create_lvm_lv(lv_name, size.size)
+      lvm_lv = parent.create_lvm_lv(lv_name, size)
```

Comment:
ok, fine. I am thinking about it, but change is so big, that I lost overview. So thanks for confirmation that it do what is expected.